executor/set.go

```diff
@@ -186,6 +186,8 @@ func (e *SetExecutor) setSysVariable(name string, v *expression.VarAssignment) e
 		if isoLevel == ast.ReadCommitted {
 			e.ctx.Txn().SetOption(kv.IsolationLevel, kv.RC)
 		}
+	} else if name == variable.TiDBDDLReorgWorkerCount {
+		domain.GetDomain(e.ctx).DDL().WorkerVars().DDLReorgWorkerCount = sessionVars.DDLReorgWorkerCount
```

Comment:
If this is a session-scope variable, we have to find the instance which is ddl-owner and run the set statement on it. We should make it a global scope variable.
(And actually, it is a server-scope variable.)